### PR TITLE
[WIP] Adding procedure on archetype review and assessment

### PR DESCRIPTION
**Pull request by mpershina**
While working on the release note about archetype assessment and review statuses ([MTA-2720](https://issues.redhat.com/browse/MTA-2720)), I noticed that the main product documentation misses the corresponding procedures. 

### Goal 

- Add the procedure on archetype assessment
- Add procedure on archetype review

### CODE REVIEW

File: docs/topics/proc_reviewing-an-archetype.adoc [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+. Open the {ProductShortName} web console.
+. Select the *Migration* view and click *Archetype*.
+. Click the *Options* menu ({kebab}) and select *Review* from the drop-down menu..
+. From the list of available questionnaires, click *Take* to select the desired assessment questionnaire.
+. In the *Assessment* menu, answer the required questions.
+. Select *Save and Review*. You will automatically be redirected to the *Review* tab.
+. Enter the following information:
+
+** *Proposed Action*: Proposed action required to complete the migration or modernization of the archetype. 
+** *Effort estimate*: The level of effort required to perform the assessment of the selected archetype.
```
Comment (nachandr) [inline docs/topics/proc_reviewing-an-archetype.adoc:27]: The level of effort required to perform the modernization/migration of the selected archetype.

Comment (mpershina) [reply docs/topics/proc_reviewing-an-archetype.adoc:27]: Ack, thank you! Adding your suggestion :-)

Comment (mpershina) [inline docs/topics/proc_reviewing-an-archetype.adoc:27]:
```suggestion
** *Effort estimate*: The level of effort required to perform the modernization or magration of the selected archetype.
```

File: docs/topics/proc_assessing-an-archetype.adoc [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+* You are logged in to an {ProductShortName} server. 	
+
+.Procedure
+
+. Open the {ProductShortName} web console.
+. Select the *Migration* view.
+. Click Archetype.
+. Click the *Options* menu ({kebab}) and select *Assess* from the drop-down menu.
+. From the list of available questionnaires, click *Take* to select the desired questionnaire questionnaire.
```
Comment (mportman12) [inline docs/topics/proc_assessing-an-archetype.adoc:22]:
```suggestion
. From the list of available questionnaires, click *Take* to select the desired questionnaire.
```

File: docs/topics/proc_assessing-an-archetype.adoc [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+NOTE: If an application is associated with several archetypes, this application is considered assessed when all associated archetypes have been assessed. 
+
+.Prerequisites
+
+* You are logged in to an {ProductShortName} server. 	
+
+.Procedure
+
+. Open the {ProductShortName} web console.
+. Select the *Migration* view.
```
Comment (mportman12) [inline docs/topics/proc_assessing-an-archetype.adoc:19]:
```suggestion
. Select the *Migration* view and click Archetype.
```

File: docs/topics/proc_assessing-an-archetype.adoc [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+. Click Archetype.
```
Comment (mportman12) [inline docs/topics/proc_assessing-an-archetype.adoc:20]:
```suggestion
```

File: docs/topics/proc_reviewing-an-archetype.adoc [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+. Open the {ProductShortName} web console.
+. Select the *Migration* view and click *Archetype*.
+. Click the *Options* menu ({kebab}) and select *Review* from the drop-down menu..
+. From the list of available questionnaires, click *Take* to select the desired assessment questionnaire.
+. In the *Assessment* menu, answer the required questions.
+. Select *Save and Review*. You will automatically be redirected to the *Review* tab.
+. Enter the following information:
+
+** *Proposed Action*: Proposed action required to complete the migration or modernization of the archetype. 
+** *Effort estimate*: The level of effort required to perform the modernization or magration of the selected archetype.
```
Comment (mportman12) [inline docs/topics/proc_reviewing-an-archetype.adoc:27]:
```suggestion
** *Effort estimate*: The level of effort required to perform the modernization or migration of the selected archetype.
```

**Review by mportman12 — COMMENTED** [review]
Left some suggestions.

**Pull request merged**